config/routes.rb

```diff
@@ -1,14 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  root 'components#index'
+  root 'components#index', via: :all
```

Comment:
This is just telling us that it can use any type of request (get, post,...), but it doesn't really do anything right? WIth or without ```via: :all``` it works the same

Reply:
It doesn't have that much of added value, I've just included it to complete all possible routes even for `POST/PUT/...` requests on the `root_path` to avoid having `Routing Error::No route matches [...] "/"`